dispatcher/pool.py

```diff
@@ -374,6 +441,10 @@ async def process_finished(self, worker, message) -> None:
         if 'timeout' in message:
             self.events.timeout_event.set()
 
+        if not self.get_unblocked_message():
+            # if nothing is in the queue, it may be clear to scale down
+            self.events.management_event.set()
```

Comment:
Had to write this to find out this is completely the wrong way to trigger scale-down. This would totally fail to scale down after getting a burst of work that goes quiet afterwards. In AWX, it scales down based on a periodic check. This should do the same thing. Just add a timeout to this:

```python
await self.events.management_event.wait(timeout=self.scale_down_check_frequency)
```

This trigger needs to be removed completely. It does no good.